src/test/ui/fn_once-moved.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0382]: use of moved value: `debug_dump_dict`
+  --> $DIR/fn_once-moved.rs:21:5
+   |
+20 |     debug_dump_dict();
+   |     --------------- value moved here
+21 |     debug_dump_dict();
+   |     ^^^^^^^^^^^^^^^ value used here after move
+   |
+   = help: closure was moved because it only implements `FnOnce`
```

Comment:
Should this point at the closure's span? If so, the text should be changed to something along the lines of

```rust
error[E0382]: use of moved value: `debug_dump_dict`
  --> $DIR/fn_once-moved.rs:21:5
15 |    let debug_dump_dict = || {
   |        --------------- closure only implements `FnOnce` so calling it moves it
...
20 |     debug_dump_dict();
   |     --------------- value moved here
21 |     debug_dump_dict();
   |     ^^^^^^^^^^^^^^^ value used here after move
```